src/components/CurrencyInputPanel/index.tsx

```diff
@@ -213,7 +214,7 @@ export default function CurrencyInputPanel({
                       ? currency.symbol.slice(0, 4) +
                         '...' +
                         currency.symbol.slice(currency.symbol.length - 5, currency.symbol.length)
-                      : currency?.symbol) || t('select Token')}
+                      : currency?.symbol) || <div data-testid="select-token-button"> {t('select Token')}</div>}
```

Comment:
Also here can we avoid using extra div?

Reply:
Same case as mentioned by Milan above, was looking for other solution but for some reason i could not get this component to work with test id

Reply:
About ids vs testids when it comes to targeting there is no big difference but i think that we, as QA's should avoid adding or editing ids to components because there may be a situation where it may affect front end(like some css styles etc.).